app/src/components/sections/TeamView/forms/AddTeamForm.vue

```diff
@@ -308,12 +299,32 @@ const searchUsers = (input: { name: string; address: string }) => {
   }
 }
 
-const onSubmit = async () => {
+const saveTeamToDatabase = async () => {
   $v.value.$touch()
   if ($v.value.$invalid) return
 
   emit('addTeam', {
-    team: teamData.value,
+    team: teamData.value
+  })
+}
+
+// Add a watch for isLoading prop
+watch(
+  () => props.isLoading,
+  (newValue, oldValue) => {
+    // If loading has finished (was true, now false)
+    if (oldValue && !newValue) {
+      // Move to next step
+      nextStep()
+    }
+  }
+)
+
```

Comment:
This watch i'm not sure that it's a good idea to use it to call next step
And for watch it's a good practice to name well the parametters
instead of oldValue and newValue use newIsloadingValue & oldIsLoadingValue
It's more explicite.

Reply:
![image](https://github.com/user-attachments/assets/7e918ae4-2213-44a7-ab3a-82776738fbd5)